src/library/scala/sys/process/ProcessBuilderImpl.scala

```diff
@@ -72,19 +73,10 @@ private[process] trait ProcessBuilderImpl {
   /** Represents a simple command without any redirection or combination. */
   private[process] class Simple(p: JProcessBuilder) extends AbstractBuilder {
     override def run(io: ProcessIO): Process = {
-      val process = try {
-        p.start() // start the external process
-      } catch {
-        case _: IndexOutOfBoundsException
-             | _: IOException
-             | _: NullPointerException
-             | _: SecurityException
-             | _: UnsupportedOperationException
-        => return FailedProcess
```

Comment:
Thought it might be helpful to have a `FailedProcess` that takes an exception, but you'd need a failed version of each "process" type. I didn't add an exception field either, it still just fails with a negative exit code.